client/modules/accounts/templates/tax-settings/taxSettings.js

```diff
@@ -0,0 +1,75 @@
+import _ from "lodash";
+import { Meteor } from "meteor/meteor";
+import { Template } from "meteor/templating";
+import { Accounts } from "/lib/collections";
+import { Accounts as AccountsSchema } from "/lib/collections/schemas/accounts";
+import { TaxEntityCodes } from "/client/collections";
+
+Template.taxSettingsPanel.helpers({
+  account() {
+    const sub = Meteor.subscribe("Accounts.single", this.member.userId);
+    if (sub.ready()) {
+      return Accounts.findOne({ _id: this.member.userId });
+    }
+    return null;
+  },
+  makeUniqueId() {
+    return `tax-settings-form-${this.member.userId}`;
+  },
+  accountsSchema() {
+    return AccountsSchema;
+  },
+  entityCodes() {
+    return _.concat(TaxEntityCodes.find().map((entityCode) => {
+      return _.assign({}, entityCode, {
+        label: entityCode.name,
+        value: entityCode.code
+      });
+    }), [{
+      label: "SET CUSTOM VALUE",
+      value: "CUSTOM USER INPUT"
+    }]);
+  }
+});
+
+Template.taxSettingsPanel.events({
+  "change [data-event-action=customType]": function (event) {
+    event.stopPropagation();
+
+    if (isCustomValue()) {
+      return $(".customerUsageType").toggleClass("hide");
+    }
+    $(".customerUsageType").addClass("hide");
+  }
+});
+
+Template.taxSettingsPanel.onCreated(function () {
+  const currentCodes = TaxEntityCodes.find().fetch();
+  if (!currentCodes.length) {
+    Meteor.call("avalara/getEntityCodes", (error, entityCodes) => {
+      _.each(entityCodes, (entityCode) => {
```

Comment:
@impactmass You could use the native `forEach` or `for (const thing of things)` rather than using a library function. Not a big deal but we generally prefer to use native function wherever possible.